qa/rpc-tests/maxuploadtarget.py

```diff
@@ -192,7 +192,8 @@ def run_test(self):
         getdata_request.inv.append(CInv(2, big_old_block))
 
         max_bytes_per_day = 200*1024*1024
-        max_bytes_available = max_bytes_per_day - 144*1000000
+        daily_buffer = 144 * 1000000 # 144 * MAX_BLOCK_SIZE
```

Comment:
Hmm.... i think either we take in the `MAX_BLOCK_SIZE` constant (controversial) or we drop the comment.